scratch/PCA/extract_data.py

```diff
@@ -0,0 +1,123 @@
+
+# coding: utf-8
+
+# In[61]:
+
+#!/usr/bin/env python
```

Comment:
This script looks very similar to `tracr/actions/pcr.py`; is one of these to be preferred over the other?